tasks/request_files/README.md

```diff
@@ -285,6 +285,7 @@ DATA
     CONFIG_JOB_ID_KEY = 'asyncOperationId'
     CONFIG_MULTIPART_CHUNKSIZE_MB_KEY = 's3MultipartChunksizeMb'
     CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY = 'orcaDefaultBucketOverride'
+    CONFIG_RESTORE_RETRIEVAL_TYPE_KEY = 'orcaRestoreTypeOverride'
```

Comment:
Consider bringing README in line with our other tasks and pointing to API document that is code generated. At a minimum create a card to update the README.

Reply:
Created https://bugs.earthdata.nasa.gov/browse/ORCA-434